src/services/config/config-service.ts

```diff
@@ -0,0 +1,77 @@
+import fs from "node:fs"
+import path from "node:path"
+import { CLIError } from "@oclif/core/lib/errors"
+
+const DEFAULT_CONFIG_PATH = "src/services/config/"
+const DEFAULT_CONFIG_FILE = "default-config.json"
+const CONFIG_FOLDER = ".ent"
+const CONFIG_FILE = "config.json"
+
+export default class ConfigService {
+  private readonly _config: any
+
+  constructor(filePath = DEFAULT_CONFIG_PATH) {
+    this._config = this.readDefaultConfigFile(filePath)
+  }
+
+  get config(): any {
+    return this._config
+  }
+
+  readDefaultConfigFile = (filePath: string) : any => {
+    const dirContents = fs.readdirSync(filePath)
+    console.log(dirContents)
+    const fileContents = fs.readFileSync(path.join(filePath, DEFAULT_CONFIG_FILE), {
+      encoding: "utf-8"
+    })
+    console.log("Default Config successfully read from disk")
+    return JSON.parse(fileContents)
+  }
+
+  writeConfigFile = (bundlePath: string, fileFolder = CONFIG_FOLDER) : void => {
+    const filePath = bundlePath.endsWith("/") ?
+      `${bundlePath}${fileFolder}` :
+      `${bundlePath}/${fileFolder}`
```

Comment:
if you want you can doit also without if and string interpolation, like this:

```javascript
const filePath = [bundlePath.split("/").filter(Boolean), fileFolder].flat().join("/")
```

Reply:
Hi @Dv92 , 
thanks for the review, I changed it with:
 ` const filePath = path.join(bundlePath, fileFolder)`
 in order to avoid issues with different operating systems.